### DISCUSSION TITLE
Group 6: Informal Entrepreneurs in Colombia

### DISCUSSION THREAD

**Discussion by bbdaniels** [gui2de/ppol768-spring23#162] [Group Projects Feedback]
Hi Group 6! Another great topic and we are thrilled to see something that blends personal situation, labor, and business policy. There are two main gaps in this proposal: Design detail and intervention detail. I will address these here so you have a basic template to follow as you expand into the next round.

As for design, I am a bit confused. How will you identify "entrepreneurs"? About how many are there? How will you produce a listing of eligible people? How are you developing a control group if "all subjects" receive the child care support? How many children are there per woman? How will it be implemented? Are there child care centers already available, or are you effectively developing a new public service? Who will provide and deliver this? Who will pay for it?

I do not quite understand how the theory of change works, or why formalization is "good" for people. Why would child care expand formalization? I think I may be misunderstanding in that "formalization" might be used here as a proxy for "full time" work. If that is the case, there needs to be a bit of rethinking -- if these women are _already_ choosing to spend time caring for children rather than working more and formalizing their businesses, what is the problem? The working assumption for the research should be that these women are already making the best choices they can given their situations, so formalization must not be ideal for them. (If there is a further upside to formalization that may not be known to women, such as social protection benefits, perhaps that would be a stronger approach, unless the goal of the project is mainly to increase tax revenues.)

To go along with that line of thinking, I would encourage you to focus back on the idea of _market failure_. A market failure problem cannot be solved by giving more resources to people (such as free childcare). Since we assume everyone is making the best choices they can in their circumstance, there has to be something else preventing them from doing what they actually want to do (we usually think of some form of mis-pricing or information asymmetry). In other words, your approach should first be to convince the reader that women _want_ to formalize their businesses (or spend more time at them, if that is the case). From there, the intervention should be slightly re-designed so that it is not just a "free stuff" intervention: for example, you could organize parent-sharing or babysitting rotations that don't provide much more resources, but "unlock" cooperative efficiencies that are unlikely to develop otherwise.

Let me know what you think and I look forward to seeing you develop the idea further!

**Comment by BeaLeydier** [comment]
Hi team,

Building off Ben's comments on your project, I think you have chosen a great topic and need to do some further digging to make your project more precise and clear.

On top of the foundataionl questions around the theory of change he outlines, I have some specific questions and comments to help you refine your project :

1. It is not very clear to me what the intervention is. Does the program already exist? Will you be providing it? How will you choose who you will be providing it to and who you will not be providing it to? Could that choice be random? If yes, what will be the challenges to implement it in practice? If not, why not, and what are the selection mechanisms that would make the two groups not comparable?

2. What is the outcome you are focusing on? If it is increasing formalization, it is also not clear to me how providing childcare does that. If it is increasing tax revenue, it is also not clear how providing child care does that. Choose one outcome -- either from the perspective of the female entrepreneurs, or from the perspective of the government, for example -- and explain what leads to the current situation of that outcome, and how your intervention would improve it. If you focus on outcomes form the women's perspective, then get rid of the language around tax revenue; and vice versa.

3. How are the two other policies you describe - VUE and information campaign - related to the intervention you propose? Are they included in it? Not included in it? Would the people you serve all be receiving other forms of formalization incentives? If yes which ones? How would they interact with yours?

4. How will you identify your study sample when collecting data? How will you be able to link that to the Ministry's record? What does it mean to enroll "randomly people on the street" and how will you follow up with them?

5. Why would a business go informal once childcare is dropped? How can a business go from being formal to becoming informal? (isn't business registration a one time thing?)

6. What are the limited resources you are mentioning about the information campaign? Who is funding the campaign? How much does it cost? How is it included in your intervention?

7. You need to harmonize and improve the language. The introduction is not very clear and starts with a reference to "these percentages" without previous context of what we are talking about, and that inconsistency is reflected throughout your pitch. The language needs to be precise, concise, clear, containing scientific references explained for the lay person. 

8. I have also noticed you have images in your folder but they aren't included in the text. Make sure you edit the MD file to include the images you intend to display!

Cheers!

**Reply by genjuro86** [reply]
Thank you for comments. 

Based on your advice, my team had meetings for an improved paper, and faced some challenges. I'd like to get help or advice regarding the challenges.

My team assumed that the reason why informal women entrepreneurs did not formally register their business was the lack of flexibility due to their child care. So, we expected if the government efficiently allocates budgets to improve formalization rate, the registration rate would rise, and as a result, tax revenue would increase. (**We thought there is a resource misallocation and this is the market failure!**) From here, the allocation of budget is from 'Information campaign' to 'child-care program'. 
 - **Information campaign**: In-person visit by a trained agent who encouraged formalization by sharing information about the potential benefits of registering and providing a detailed explanation of the process and required documentation.
 - **Child-care program**: 100 percent free daycare and preschool for 300 children and 240 families in this area, serving children ages 4 months to 4 years old.
 
However, the challenge was **whether the cost of information campaigns is comparable to child-care support in terms of redistribution of resources. It was actually difficult to get accurate information on the budget amount between the two programs, so I doubt whether the two budget levels are similar.** If we have to spend too much money on child-care program, we will face the resource constraint issue. Furthermore, I couldn't find other comparable control groups (Government programs which have been provided to improve formalization rate). In this situation, my team is trying to decide the best course for our project but we need help how to make it more reasonable. So, it would be very grateful if you can help my team.

**Reply by bbdaniels** [reply]
We are happy to meet to discuss with your team at any of our ordinarily available meeting times. Please reach out to book a slot as indicated on the syllabus. Thanks and hope you've had a great break!

**Comment by bbdaniels** [comment]
# Concept Note Review

Hi all, I am not going to go into as much detail here as I have with the other groups because we just had a very detailed conversation, but I will highlight the key points here. As I said, I think you can make this work, but you need to hone in on what you are trying to do. I will note that in our conversation you were (a) not sure who the beneficiaries were (government or women) and (b) not clear on the failure we are trying to resolve. We discussed changing the focus to solving a coordination and commitment problem for child-care among informal women workers in Colombia. If you take that idea forward and build it out so you have clear ideas of the structure of possible interventions (universe, sample, strata, clusters), you will have something to work with.

We further discussed identifying effects by number of children, and by observing data directly on hours worked. We also discussed surveys for income, etc. as key outcomes. I think the question of formalization as an outcome was still left open, since we did not reach a firm conclusion of whether women actually wanted to formalize and whether there are barriers there, or whether we should rely on the observed choice to not formalize as a revealed preference. You should include that discussion and your conclusion as well as the new experimental design in your final version, You have done a great job setting up the context and understanding the environment, and you just need to make the next step into fully fleshing out the intervention proposal and its goal to have working materials. Then you can do power calculations and direct simulations along the lines we discuss in my lectures. Keep at it!
